src/client/renderingengine.cpp

```diff
@@ -463,15 +471,11 @@ const VideoDriverInfo &RenderingEngine::getVideoDriverInfo(irr::video::E_DRIVER_
 float RenderingEngine::getDisplayDensity()
 {
 #ifndef __ANDROID__
-	static float cached_display_density = [&] {
-		float dpi = get_raw_device()->getDisplayDensity();
-		// fall back to manually specified dpi
-		if (dpi == 0.0f)
-			dpi = g_settings->getFloat("screen_dpi");
-		return dpi / 96.0f;
-	}();
-	return std::max(cached_display_density * g_settings->getFloat("display_density_factor"), 0.5f);
-
+	float dpi = get_raw_device()->getDisplayDensity();
+	// fall back to manually specified dpi
+	if (dpi == 0.0f)
+		dpi = g_settings->getFloat("screen_dpi");
```

Comment:
I think this setting is outdated these days because dpi should be available everywhere.
it's not even consistently applied either.

Reply:
well, I can certainly remove it 🤷